torch/_inductor/lowering.py

```diff
@@ -4327,57 +4327,62 @@ def pooling_size(x, i, kernel_size, stride, padding, ceil_mode, *, dilation=None
     return x_out, ceil_mode
 
 
-def should_fallback_max_pool2d_with_indices(kernel_size):
-    kernel_size = pad_listlike(kernel_size, 2)
-    window_size = kernel_size[0] * kernel_size[1]
+def should_fallback_max_pool_with_indices(kernel_size, *, dim):
```

Comment:
nit: n_dim, since this is not specifying the particular dim, it's specifying number (which is different than most of the other functions here)

Reply:
Renamed to `n_dim`